chapter_convolutional-modern/googlenet_vn.md

```diff
@@ -296,7 +302,14 @@ The ratio of the number of channels assigned in the Inception block is obtained
 3. Why do we need a large range convolution initially?
 -->
 
-*dịch đoạn phía trên*
+1. Có nhiều biến thể của mô hình GoogLeNet. Hãy thử lập trình và chạy chúng. Một số biến thể bao gồm:
+    * Thêm vào một tầng chuẩn hoá theo batch :cite:`Ioffe.Szegedy.2015`, như đã được mô tả ở phần :numref:`sec_batch_norm`.
+    * Thực hiện chỉnh sửa khối Inception theo :cite:`Szegedy.Vanhoucke.Ioffe.ea.2016`.
+    * Sử dụng kỹ thuật 'label smoothing' cho sự điều chuẩn hoá mô hình
+    * Thêm khối Inception vào trong kết nối thặng dư :cite:`Szegedy.Ioffe.Vanhoucke.ea.2017`, như mô tả sau ở phần :numref:`sec_resnet`.
+1. Kích thước tối thiểu của hình ảnh đầu vào để mô hình GoogLeNet hoạt động là bao nhiêu?
+2. So sánh số lượng tham số của mô hình của AlexNet, VGG và NiN với GoogLeNet. Làm thế nào mà hai kiến trúc mạng NiN và GoogLeNet lại giảm đáng kể về số lượng tham số? 
+3. Tại sao chúng ta cần nhiều tầng tích chập với kích thước vùng tiếp nhận khác nhau?
```

Comment:
```suggestion
3. Tại sao chúng ta cần tầng tích chập kích thước lớn ở đầu mạng?
```

Reply:
@cuongvng cái này mình nghĩ `large range` ở đây không phải là `lớn`, `large range` ở đây có nghĩa là `có nhiều loại kích thước của các cửa sổ trượt `

Reply:
hm về ngữ pháp thì nếu theo nghĩa `nhiều loại kích thước cửa sổ trượt` thì phải viết là `a large range of convolutions`. Với cả còn `initially` bị bỏ đi nữa. Tạm thời cứ chờ reviewers khác cho ý kiến xem.

Reply:
@cuongvng cái này bạn chắc đúng rồi á, mình xem lại thấy ban đầu nó xài conv2d 7x7